test/jdk/java/awt/List/ActionEventTest/ActionEventTest.java

```diff
@@ -70,9 +71,9 @@ public void actionPerformed(ActionEvent ae) {
 
                 if ((md & expectedMask) != expectedMask) {
 
-                    robot.keyRelease(KeyEvent.VK_ALT);
-                    robot.keyRelease(KeyEvent.VK_SHIFT);
                     robot.keyRelease(KeyEvent.VK_CONTROL);
+                    robot.keyRelease(KeyEvent.VK_SHIFT);
+                    robot.keyRelease(KeyEvent.VK_ALT);
                     dispose();
                     throw new RuntimeException("Action Event modifiers are not"
                         + " set correctly.");
```

Comment:
This exception should be thrown from the main thread. An action listener is called on EDT, thus the exception won't fail the test, will it?

And using robot to release keys from EDT as part of the clean-up doesn't seem right.

Reply:
> And using robot to release keys from EDT as part of the clean-up doesn't seem right.

I am just rectifying the order of the key release..It was pressed in order ALT SHIFT CONTROL and the release was ALT SHIFT CONTROL.. which I rectified to be release in order CONTROL SHIFT ALT